src/PharoThings-Hardware-Core.package/PotBoard.class/instance/addExistingConnector.with..st

```diff
@@ -0,0 +1,8 @@
+initialization
+addExistingConnector: aString with: pinsArray
+	| connector pinsArrayExisting |
+	pinsArrayExisting  := (connectors at: 1) pins.
+	connector := PotBoardConnector named: aString in: self with: pinsArrayExisting , pinsArray.
+	connectors removeAll.
```

Comment:
This removeAll will remove any other connectors. It looks like a bug.

Reply:
If I don't remove the connector before (created by superclass RpiBoardB), will be created a new array inside "connectors" and will create a new tab on the inspector (like P5 of RpiBoardBRev2). But before remove, I copy the array to concatenate with the new pinsArray. 
I did some tests saving the image with the LEDs on and when I reboot the Raspberry and start Pharo again, the LEDs return to the last state (on).
I changed to removeFirst, it is ok? Or should not have this anyway? I'm telling more in next answer.